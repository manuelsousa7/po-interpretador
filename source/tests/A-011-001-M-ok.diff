*** tests/A-011-001-M-ok.out	Mon Dec  5 13:13:09 2016
--- tests/A-011-001-M-ok.outhyp	Tue Dec  6 13:19:51 2016
***************
*** 31,45 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 1
! 2
! 3
! 4
! 6
! 7
! 8
! 9
! Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 31,37 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
***************
*** 47,53 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Posição do programa: Expressão: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
--- 39,45 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Listar programa
  2 - Executar programa
  3 - Adicionar expressão ao programa
***************
*** 55,131 ****
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: (print 0)
! (print 2)
! (print 3)
! (print 4)
! (print 6)
! (print 7)
! (print 8)
! (print 9)
! Menu de Edição
! 1 - Listar programa
! 2 - Executar programa
! 3 - Adicionar expressão ao programa
! 4 - Substituir expressão no programa
! 5 - Mostrar os identificadores presentes no programa
! 6 - Mostrar os identificadores do programa sem inicialização explícita
! 0 - Sair
! Escolha uma opção: 0
! 2
! 3
! 4
! 6
! 7
! 8
! 9
! Menu de Edição
! 1 - Listar programa
! 2 - Executar programa
! 3 - Adicionar expressão ao programa
! 4 - Substituir expressão no programa
! 5 - Mostrar os identificadores presentes no programa
! 6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Posição do programa: Expressão: Menu de Edição
! 1 - Listar programa
! 2 - Executar programa
! 3 - Adicionar expressão ao programa
! 4 - Substituir expressão no programa
! 5 - Mostrar os identificadores presentes no programa
! 6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: (print 0)
! (print 2)
! (print 3)
! (print 4)
! (print 6)
! (print 7)
! (print 8)
! (print "nove")
! Menu de Edição
! 1 - Listar programa
! 2 - Executar programa
! 3 - Adicionar expressão ao programa
! 4 - Substituir expressão no programa
! 5 - Mostrar os identificadores presentes no programa
! 6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: 0
! 2
! 3
! 4
! 6
! 7
! 8
! nove
! Menu de Edição
! 1 - Listar programa
! 2 - Executar programa
! 3 - Adicionar expressão ao programa
! 4 - Substituir expressão no programa
! 5 - Mostrar os identificadores presentes no programa
! 6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Criar
--- 47,90 ----
  5 - Mostrar os identificadores presentes no programa
  6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Criar
! 2 - Abrir
! 3 - Guardar
! 4 - Criar Programa
! 5 - Ler Programa
! 6 - Escrever Programa
! 7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Menu Principal
! 1 - Criar
! 2 - Abrir
! 3 - Guardar
! 4 - Criar Programa
! 5 - Ler Programa
! 6 - Escrever Programa
! 7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: O ficheiro '4' não existe.
! Menu Principal
! 1 - Criar
! 2 - Abrir
! 3 - Guardar
! 4 - Criar Programa
! 5 - Ler Programa
! 6 - Escrever Programa
! 7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Nome do program: O programa (print "nove") não existe.
! Menu Principal
! 1 - Criar
! 2 - Abrir
! 3 - Guardar
! 4 - Criar Programa
! 5 - Ler Programa
! 6 - Escrever Programa
! 7 - Manipular Programa
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Criar
***************
*** 136,139 ****
--- 95,108 ----
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
+ Escolha uma opção: Ficheiro a abrir: O ficheiro '0' não existe.
+ Menu Principal
+ 1 - Criar
+ 2 - Abrir
+ 3 - Guardar
+ 4 - Criar Programa
+ 5 - Ler Programa
+ 6 - Escrever Programa
+ 7 - Manipular Programa
+ 0 - Sair
  Escolha uma opção: 
\ No newline at end of file
