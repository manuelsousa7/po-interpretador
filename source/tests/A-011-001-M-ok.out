Menu Principal
1 - Criar
2 - Abrir
3 - Guardar
4 - Criar Programa
5 - Ler Programa
6 - Escrever Programa
7 - Manipular Programa
0 - Sair
Escolha uma opção: Nome do program: Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: (print 1)
(print 2)
(print 3)
(print 4)
(print 6)
(print 7)
(print 8)
(print 9)
Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: 1
2
3
4
6
7
8
9
Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: Posição do programa: Expressão: Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: (print 0)
(print 2)
(print 3)
(print 4)
(print 6)
(print 7)
(print 8)
(print 9)
Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: 0
2
3
4
6
7
8
9
Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: Posição do programa: Expressão: Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: (print 0)
(print 2)
(print 3)
(print 4)
(print 6)
(print 7)
(print 8)
(print "nove")
Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: 0
2
3
4
6
7
8
nove
Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: Menu Principal
1 - Criar
2 - Abrir
3 - Guardar
4 - Criar Programa
5 - Ler Programa
6 - Escrever Programa
7 - Manipular Programa
0 - Sair
Escolha uma opção: 